helpers/pdfUtils.py

```diff
@@ -0,0 +1,20 @@
+import logging
+import tempfile
+import subprocess
+from django.conf import settings
+from subprocess import CalledProcessError
+
+logger = logging.getLogger(__name__)
+
+SCRIPT_PATH = settings.BASE_DIR + "/bin/generate-pdf.js" 
+
+def get_pdf_for_url(url):
+  try:
+    tmp = tempfile.NamedTemporaryFile(delete=True)
+    child = subprocess.Popen(["node", SCRIPT_PATH, "-w", tmp.name, "-u %s" % (url)], stdout=subprocess.PIPE)
+    child.communicate()
+    contents = tmp.read()
+    tmp.close()
```

Comment:
Should use a python `with` block instead - it's quite nice! Python's version of Java try with resources (though I think it came earlier?)

Reply:
sounds good